cache/spring-cloud-azure-starter/spring-cloud-azure-sample-cache/README.md

```diff
@@ -20,7 +20,7 @@ You will build an application using the Spring Boot Starter Redis, Spring Cloud
 This sample will create Azure resources using Terraform. If you choose to run it without using Terraform to provision resources, please pay attention to:
 > [!IMPORTANT]  
 > If you choose to use a security principal to authenticate and authorize with Azure Active Directory for accessing an Azure resource
-> please refer to [Authorize access with Azure AD](https://microsoft.github.io/spring-cloud-azure/docs/current/reference/html/index.html#authorize-access-with-azure-active-directory) to make sure the security principal has been granted the sufficient permission to access the Azure resource.
+> please refer to [Authorize access with Azure AD](https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html#authorize-access-with-azure-active-directory) to make sure the security principal has been granted the sufficient permission to access the Azure resource.
```

Comment:
`current` -> `4.0.0-beta.3/4.0.0-beta.3`

Reply:
This is in the template, I assume this link is not branch-related, and should always link to the current link.